src/cinder/audio/OutputNode.cpp

```diff
@@ -96,7 +98,21 @@ OutputDeviceNode::OutputDeviceNode( const DeviceRef &device, const Format &forma
 	mDidChangeConn = mDevice->getSignalParamsDidChange().connect( bind( &OutputDeviceNode::deviceParamsDidChange, this ) );
 
 	mInterruptionBeganConn = Context::deviceManager()->getSignalInterruptionBegan().connect( [this] { disable(); } );
-	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool shouldResume ) { enable(); } );
+	mInterruptionEndedConn = Context::deviceManager()->getSignalInterruptionEnded().connect( [this]( bool resumeImmediately ) {
```

Comment:
The only time there won't be an App will be during `prepareSettings`, I think it is fair to say that you can't be manipulating the audio graph until after at least your app's constructor has been called. So I think just registering for the app became active signal once from the DeviceManagerAudioSession's constructor is OK. Actually I'm already doing the same for the shutdown signal.

I also like that user code becomes simpler this way, not having to worry about what that bool means and whether or not functionality should change based on it.